source/administration/plugins.rst

```diff
@@ -4,7 +4,7 @@ Plugins (Beta)
 .. note::
   This is the admin documentation for plugins. If you're a developer looking to build a plugin, see `our developer documentation <https://developers.mattermost.com/extend/plugins>`_.
 
-Mattermost supports plugins, which allow users to customize and extend the platform by adding specific features.
+Mattermost supports plugins, which allow users to customize and extend the platform by adding specific features. See our `demo plugin <https://github.com/mattermost/mattermost-plugin-demo>`_ that demonstrates the capabilities of a Mattermost plugin. For a starting point to write a Mattermost plugin, see our `sample plugin <https://github.com/mattermost/mattermost-plugin-demo>`_.
 
```

Comment:
These 2 links are the same, is this expected or can the sentences be combined?

Reply:
Good catch, fixed